public/i18n/en/translation.json

```diff
@@ -517,13 +517,14 @@
     },
     "storage": {
       "title": "Storage",
-      "description": "You can keep a copy of your system logs on an external storage device, such as a USB stick. This can be useful for troubleshooting problems or for keeping a record of your system activity.",
-      "format_configure_storage": "Format and configure storage",
+      "description": "You can keep a copy of your system logs on an persistent storage such as a USB stick or a SATA drive. This can be useful for troubleshooting problems or for keeping a record of your system activity. If you lack an additional drive, you can alternatively choose the operating system disk if it contains available unpartitioned space; the system will automatically generate a partition and utilize it.",
+      "format_configure_storage": "Configure storage",
       "configure_storage": "Configure storage",
       "format_storage_warning": "All data contained in '{device}' will be erased",
-      "configure_storage_logs_description": "From now on logs will also be written on the device at mount point",
+      "new_partition": "A new partition will be created on device '{device}'",
+      "configure_storage_logs_description": "From now on, logs will also be written on the device at the path",
       "configure_storage_extra_data_description": "Extra data can be automatically stored to the device once a day",
-      "format_and_configure": "Format and configure",
+      "format_and_configure": "Configure",
```

Comment:
If the user selects an external storage, it's important to highlight that existing data will be erased. Can we show "Format and configure" on the confirmation button if an external storage was selected?

Reply:
The information is already present inside the modal.

Reply:
Added also inside the button, even if redundant.